### ISSUE TITLE
Find a theme song

### ISSUE THREAD

**Issue by chriseidhof** [BananaKit/BananaKit#4]

**Comment by dbgrandi** [comment]
Good call. Everybody has a logo, but I think more projects should have a theme song.

Possible options:
- [BananaKit](https://soundcloud.com/theofficialmonopoly/bananakit) by The Official Mono/Poly
- Anything by Bananarama (e.g. [Cruel Summer](https://www.youtube.com/watch?v=9ePIZugahFc))
- [Banana Boat Song (Day-O)](http://open.spotify.com/track/4fHDlIntTsRGSyTg5UYZYC)

**Comment by paulyoung** [comment]
[Bananas in Pajamas](https://www.youtube.com/watch?v=M1iu2uGtlBw)
